### PR TITLE
SC-455: Define the current environment in config

### PR DESCRIPTION
**Pull request by alexanderM91**
Developer(s): @alexanderM91 

- Ticket: https://spryker.atlassian.net/browse/SC-455

- Academy PR: ACADEMY_URL_HERE

- Release Group: https://release.spryker.com/release-groups/view/1457

- Core PR:  https://github.com/spryker/spryker/pull/5500
- Shop PR: https://github.com/spryker/spryker-shop/pull/1000
- Suite non-split PR: https://github.com/spryker/suite-nonsplit/pull/2312

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   SprykGui                | patch (currently 0.x)                 |                     |
   

#### Release Notes

Application behavior is controlled via explicit configuration settings instead of environment names.

-----------------------------------------

##### Change log

- Introduced `SprykGuiConfig::isSprykWebInterfaceEnabled()` to enable/disable Spryk web interface.

### CODE REVIEW

**Comment by dereuromark** [comment]
Please remove leading \ from changelogs FQCN as per specs.

File: src/SprykerSdk/Zed/SprykGui/SprykGuiConfig.php [hunk 1]
```diff
@@ -18,14 +17,22 @@ class SprykGuiConfig extends AbstractBundleConfig
...
     public function getProjectNamespaces(): array
     {
-        return Config::get(KernelConstants::PROJECT_NAMESPACES, []);
+        return $this->get(KernelConstants::PROJECT_NAMESPACES, []);
+    }
+
+    /**
+     * @return bool
+     */
+    public function isSprykWebInterfaceEnabled(): bool
```
Comment (stereomon) [inline src/SprykerSdk/Zed/SprykGui/SprykGuiConfig.php]: IMO this check is not needed as Spryks should only be installed in dev mode and we do not have this kind of checks anywhere else.

Comment (dereuromark) [reply src/SprykerSdk/Zed/SprykGui/SprykGuiConfig.php]: I agree with you. The requirement for this at the time came from the product side.
I was also always against it :) But now that we have it, it shouldnt hurt to have either.

Comment (sprymiker) [reply src/SprykerSdk/Zed/SprykGui/SprykGuiConfig.php]: Only in the sake of BC.

Comment (sprymiker) [reply src/SprykerSdk/Zed/SprykGui/SprykGuiConfig.php]: But at least we are giving the way to override it instead of hardcode.

**Comment by sprymiker** [comment]
@alexanderM91 Please add release table. I have no possibility to edit the description of foreign PR.

**Pull request merged**